src/style.css

```diff
@@ -1,3 +1,72 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+  padding: 0;
   margin: 0;
+  justify-content: flex-end;
+}
+
+.nav__item {
+  display: inline-block;
+  padding: 4px;
+  margin-right: 20;
```

Comment:
The 'margin-right' property is missing a unit. It should be 'margin-right: 20px;' to ensure proper spacing.